src/api/v4/dataview/category.js

```diff
@@ -0,0 +1,241 @@
+var _ = require('underscore');
+var Base = require('./base');
+var constants = require('../constants');
+var CategoryDataviewModel = require('../../../dataviews/category-dataview-model');
+var CategoryFilter = require('../../../windshaft/filters/category');
+
+/**
+ * Category dataview object
+ *
+ * @param {carto.source.Base} source - The source where the dataview will fetch the data.
+ * @param {string} column - The column name to get the data.
+ * @param {object} options
+ * @param {number} options.maxCategories - The maximum number of categories in the response.
```

Comment:
`options.limit` might also work here.

Reply:
:+1:

Reply:
limit is other option for maxCategories, yes